index.html

```diff
@@ -232,7 +233,7 @@ <h3 class="connect-text">LET'S CONNECT</h3>
           /></a>
         </li>
       </ul>
-      <button class="about-button">Get my resume</button>
+      <button class="about-button">Get my resume,hire me</button>
```

Comment:
The button you implemented seems larger than that of the Figma template. Please check and use the template to fix it